src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public abstract class Color {
+    public String color;
```

Comment:
Class fields should be private and accessed through getters and setters to encapsulate the data and maintain control over the class state. Moreover, if `Color` is intended to be an enum or a set of constants, this field should not be here at all.